Extending/Packages/Creating-a-nuget-package/index.md

```diff
@@ -0,0 +1,79 @@
+
+You can also create NuGet packages manually using the [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer), although this method is harder to automate than using the command line tools. 
+
+## Create a `.nuspec` file
+
+A `.nuspec` file is an xml file that tells NuGet how to create a NuGet package. It contains basic information about a package, and dependencies that your package might require and a list of files to include as part of the NuGet package.
+
+The [full specification of a nuspec file](https://docs.microsoft.com/en-us/nuget/reference/nuspec) can be found within the NuGet documentation. 
+
+Below is an example `.nuspec` file that might be used to create a package for the [Creating a Custom Dashboard tutorial](../../../Tutorials/Creating-a-Custom-Dashboard/index.md). 
+
+```xml
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+  <metadata>
+    <id>Custom Welcome dashboard</id>
+    <version>1.0.0</version>
+    <authors>Umbraco User</authors>
+    <owners>Umbraco user</owners>
+    <projectUrl>https://github.com/umbraco/customwelcomedashboard</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>This will add a dashboard to your content section</description>
+    <releaseNotes><![CDATA[1.0.0 Initial release]]></releaseNotes>
+    <copyright>Copyright 2019</copyright>
+    <tags>umbraco, umbraco-cms</tags>
+    <dependencies>
+      <dependency id="WindowsAzure.ServiceBus" version="6.0.0" />
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="bin\release\CustomWelcomeDashboard.dll" target="lib\net462" />
+    <file src="App_Plugins\CustomWelcomeDashboard\**\*.*" target="content\App_Plugins\CustomWelcomeDashboard" />
+  </files>
+</package>
+```
+
+:::note 
+Unlike an Umbraco Package, a NuGet package can reference other packages your code might be dependent on, as a rule you don't need to include packages that are part of the main Umbraco installation.
+:::
+
+## Creating the package 
+Once you have a .nuspec file you can use the NuGet CLI to create the NuGet package. 
+
+An example command to create your package may look like this:
+```
+nuget pack myCustomWelcome.nuspec -version 1.0.0 
+```
+
+There are multiple arguments you can pass on the command line to the NuGet command [more information in the NuGet documentation](https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference). 
+
+The `-version` parameter allows you to replace the version number in the nuspec file with the one specified, this means you can build new versions of your package without having to change the .nuspec file each time.
+
+:::note
+Once you have successfully ran the nuget pack command you will have a file in the format `PackageName.VersionNumber.nupkg`. This is your nuget package file. 
```

Comment:
Is it on purpose that you're writing "nuget" here, instead of "NuGet"? 🤔

Reply:
its the exe rather than the name it should probibly be ..  `nuget pack` command ... other one NuGet package

Reply:
Ah, makes sense! 😁